### DISCUSSION TITLE
Visualizing pore/throat occupancy

### DISCUSSION THREAD

**Discussion by mabdelr** [PMEAL/OpenPNM#2329] [Q&A] [answered]
Hi there,

I was wondering if there was to map the pore/throat fluid occupancy outputted from percolation algorithms back to .tif style format? I have a few CT stacks that I've performed simulations on using OpenPNM, and now I'd like to visualize the solution outputs of these simulations along with the original .tif stacks in ParaView.

**Comment by jgostick** [comment, marked as answer]
Assumiung the network is extracted using snow, so that you have an image with each pore region labelled, then yes its possible.  The extracted network has a 'pore.region_label' array, which you can use to index into the pore array of interest.  You just need to remember that openpnm indices start at 0, while the regions start at 0.  So:

```
im = net['pore.equivalent_diameter'][snow_n.regions-1]
```

Of course you can replace 'pore.equivalent_diameter' with 'pore.occupancy' or whatever you like.
